src/shared/common/Button.js

```diff
@@ -0,0 +1,83 @@
+// @flow
+import * as React from 'react';
+
+import { Component } from 'react';
+import { Text, View, StyleSheet, Pressable } from 'react-native';
+
+// loosely adapted from https://github.com/APSL/react-native-button
+const styles = StyleSheet.create({
+    button: {
+        height: 44,
+        flexDirection: 'row',
+        alignItems: 'center',
+        borderWidth: 1,
+        borderRadius: 8,
+        marginBottom: 10,
+        alignSelf: 'stretch',
+        justifyContent: 'center',
+    },
+    text: {
+        fontSize: 16,
+        lineHeight: 21,
+        fontWeight: 'bold',
+        letterSpacing: 0.25,
+        color: 'white',
+    },
+    opacity: {
+        opacity: 0.5,
+    },
+});
+
+type Props = {
+    children: any,
+    onPress: () => any,
```

Comment:
```suggestion
    onPress: () => void,
```
I think this is the correct type. But you might want to double check elsewhere in the codebase, or in the docs https://flow.org/en/docs/react/types/